app/services/copy_incident_application_to_new_account.rb

```diff
@@ -0,0 +1,62 @@
+class CopyIncidentApplicationToNewAccount
+  attr_reader :original_application_form, :candidate_email_address
+
+  def initialize(original_application_form:, candidate_email_address:)
+    @original_application_form = original_application_form
+    @candidate_email_address = candidate_email_address
+  end
+
+  def call!
+    # Create a new Candidate
+    # this will not create a duplicate if they already exist
+    log('Creating a new candidate if they do not exist')
+    CandidateInterface::SignUpForm.new(email_address: candidate_email_address).save
+    candidate = Candidate.find_by!(email_address: candidate_email_address)
```

Comment:
The email address has to be in all lowercase here 🤦

Reply:
Done